src/main/webapp/index.html

```diff
@@ -22,7 +22,14 @@ <h1>
           Discover the world's hidden gems <br />
           before planning your next trip.
         </h1>
-        <input type="text" name="search" placeholder="Search a city, country, etc." />
+        
+        <!-- Search Bar -->
+        <form autocomplete="off">
+          <div class="autocomplete">
+            <input id="myInput" type="text" name="Country" placeholder="Search a country" />
```

Comment:
I'm going to suggest having the autocompletion `<div>` already present here so you're not creating and deleting it all the time in JavaScript:
```suggestion
            <input id="myInput" type="text" name="q" placeholder="Search a country">
              <div id="autocomplete-list" class="autocomplete-items"></div>
            </input>
```